app/scripts/modules/core/src/presentation/main.less

```diff
@@ -62,7 +62,7 @@ html {
     }
     color: var(--color-accent);
     &:active, &:hover, &:focus, &.active {
-      color: var(--color-alto);
+      color: var(--color-primary);
```

Comment:
sorry :(

Reply:
No worries - we knew there would be some little ones like this.